lib/const.rb

```diff
@@ -202,13 +206,12 @@
 end
 
 CREW_LINKER ||= ENV.fetch('CREW_LINKER', 'mold') unless defined?(CREW_LINKER)
-CREW_GLIBC_OVERRIDE_LINKER_FLAGS ||= ARCH == 'x86_64' && LIBC_VERSION.to_f >= 2.35 ? " #{File.join(CREW_LIB_PREFIX, 'libC.so.6')} " : ''
-CREW_LINKER_FLAGS ||= ENV.fetch('CREW_LINKER_FLAGS', CREW_GLIBC_OVERRIDE_LINKER_FLAGS) unless defined?(CREW_LINKER_FLAGS)
+CREW_LINKER_FLAGS ||= ENV.fetch('CREW_LINKER_FLAGS', '') unless defined?(CREW_LINKER_FLAGS)
 
 CREW_CORE_FLAGS           ||= "-O3 -pipe -ffat-lto-objects -fPIC #{CREW_ARCH_FLAGS} -fuse-ld=#{CREW_LINKER} #{CREW_LINKER_FLAGS}"
 CREW_COMMON_FLAGS         ||= "#{CREW_CORE_FLAGS} -flto=auto"
 CREW_COMMON_FNO_LTO_FLAGS ||= "#{CREW_CORE_FLAGS} -fno-lto"
-CREW_LDFLAGS              ||= "-flto=auto #{CREW_LINKER_FLAGS}"
+CREW_LDFLAGS              ||= "#{CREW_LINKER_FLAGS} -Wl,--dynamic-linker,#{CREW_GLIBC_INTERPRETER}"
```

Comment:
Trailing `,` if `CREW_GLIBC_INTERPRETER` is nil?

Reply:
You want this:
```ruby
CREW_LDFLAGS ||= "#{CREW_LINKER_FLAGS} #{CREW_GLIBC_INTERPRETER.nil? ? '' : "-Wl,--dynamic-linker,#{CREW_GLIBC_INTERPRETER}"}"
```